src/main/resources/Ideas/IdeasSheet.xml

```diff
@@ -356,9 +372,48 @@
   color: $theme.linkColor;
   cursor: pointer;
   text-decoration: underline;
+}
+
+.recent {
+  line-height: 30px;
+}
+
+.recent .xvote-pro, .recent .xvote-con {
+  margin: .1em .3em .5em 0;
+  width: 15px;
+  height: 15px;
+  line-height: 15px;
+  font-size: 15px;
+  padding: 1px;
+  border-radius: 0;
+  font-size: .8em;
+}
+
+.recent .noitems {
+  font-size: inherit;
+}
+
+.ideas-recent {
+  overflow: hidden;
+  padding: 0 !important;
+}
+
+.ideas-recent .recent {
+  height: 100%;
+  width: 100% !important;
+  list-style-type: none;
+  padding: 0.1em 0;
+}
+
+.ideas-recent .recent:last-of-type {
+  margin-top: 10px;
+  line-height: initial;
```

Comment:
Yes, because otherwise there would be a bigger distance from all the entries and the link "See all ideas". See: 
![macro](https://cloud.githubusercontent.com/assets/8993426/5875187/179184fe-a313-11e4-9ab0-75ab8a535add.png)